factorial/rust/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "wasmcloud-interface-factorial"
-version = "0.4.0"
+version = "0.4.1"
```

Comment:
I think this should be a minor version bump for all interfaces, the bump of `wasmbus-rpc` `0.7` -> `0.8.2` will cause actors that use this interface and an older version of wasmbus-rpc to fail to compile

Reply:
I think it needs to be semver minor bump - too many cross-dependencies. An actor will only have an issue if they run `cargo update`, but if they do then we should have updated wasmbus-rpc and the applicable interfaces by then.

Reply:
sorry - misread your response - yes all interfaces get a semver minor bump. The last published crate for this interface was 0.3.x.